spotlight-client/src/SiteFooter/SiteFooter.tsx

```diff
@@ -72,6 +71,15 @@ const CoBranding = styled.div`
   width: 100%;
 `;
 
+const DocButton = styled.a`
+  background: ${colors.caption};
+  color: ${colors.footerBackground} !important;
```

Comment:
Curious what's the reason for using `!important` here?

Reply:
That's interesting, I hadn't worked much in CSS before and looking into this, it's my understanding that this is because the wrapper's `.Wrapper a {}` css selector is more specific than the DocButton's `.DocButton`  selector, so the Wrapper overrides DocButton's `a` styles, is that right?